pkg/reconciler/buildrun/resources/conditions.go

```diff
@@ -105,19 +105,29 @@ func UpdateBuildRunUsingTaskRunCondition(ctx context.Context, client client.Clie
 				message = pod.Status.Message
 				reason = buildv1beta1.BuildRunStatePodEvicted
 				if failedContainer != nil {
-					//nolint:staticcheck // SA1019 we want to give users some time to adopt to failureDetails
 					buildRun.Status.FailureDetails.Location.Container = failedContainer.Name
 				}
 			} else if failedContainer != nil {
-				//nolint:staticcheck // SA1019 we want to give users some time to adopt to failureDetails
 				buildRun.Status.FailureDetails.Location.Container = failedContainer.Name
-				message = fmt.Sprintf("buildrun step %s failed in pod %s, for detailed information: kubectl --namespace %s logs %s --container=%s",
-					failedContainer.Name,
-					pod.Name,
-					pod.Namespace,
-					pod.Name,
-					failedContainer.Name,
-				)
+
+				if failedContainerStatus != nil && failedContainerStatus.State.Terminated != nil && failedContainerStatus.State.Terminated.Reason == "OOMKilled" {
+					reason = buildv1beta1.BuildRunStateStepOutOfMemory
+					message = fmt.Sprintf("buildrun step %s failed due to out-of-memory in pod %s, for detailed information: kubectl --namespace %s logs %s --container=%s",
```

Comment:
```suggestion
					message = fmt.Sprintf("buildrun pod %s failed in step %s due to out-of-memory, for detailed information: kubectl --namespace %s logs %s --container=%s",
```
and then adjust your arguments position in line 116

Reply:
That proposal would mean that I also change the message for non-oom failures in the same way. Though that message is like that for years now. Should we really make this change?

Reply:
As discussed in slack, I removed the pod name from the message as it is not of primary relevance for the end user. It is still available in the `kubectl logs` command and the BuildRun status.